packages/ifos/src/utils/getIfoConfig.ts

```diff
@@ -60,7 +60,7 @@ export async function getInActiveIfos(chainId?: ChainId): Promise<Ifo[]> {
 export async function getTotalIFOSold(chainId?: ChainId): Promise<number> {
   const ifos = await getIfoConfig(chainId)
   const unwrap = (usd: string) => {
-    return Number(usd.replace(/[$,.]/g, ''))
+    return Number(usd.replace(/[$,]/g, ''))
```

Comment:
my bad. Btw are we using locales for the formatted number? I'm wondering if `,` and `.` would reverse for some countries